### PR TITLE
Dotcom contact page

### PR DESCRIPTION
**Pull request by rupertbates**
## Why are you doing this?
During the Corona virus outbreak we may need to change our contact details at short notice. 
this PR uses a new switch `useDotcomContactPage' which when enabled will change all the contact us information to point to a single page on dotcom which can be easily updated by central production.

[**Trello Card**](https://trello.com/c/dwy3Ir7I/2925-update-contact-us-details-to-point-to-a-webpage-in-event-of-customer-service-outage)

## TODO
 - [x] Fix layout issue on digisub page
 - [x] Contributions page?
 - [ ] Deploy new switch

### CODE REVIEW

File: support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ class CatalogServiceIntegrationSpec extends AsyncFlatSpec with Matchers with Ins
     forAll(product.ratePlans(environment))(
       ratePlan =>
         service.getPriceList(ratePlan).fold {
-          Console.println(s"Failed to find a catalog price list for $environment > $product > ${ratePlan.id}")
+          Console.println(s"Failed to find a catalog price list for $environment > $product > ${ratePlan.billingPeriod} > ${ratePlan.id}")
```
Comment (jfsoul) [inline support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala]: Unrelated?

File: support-frontend/assets/components/legal/contribLegal/contribLegal.jsx [hunk 2]
```diff
@@ -19,7 +23,8 @@ type PropTypes = {|
 // ----- Component ----- //
 
 export default function ContribLegal(props: PropTypes) {
-
+  const contactUs = useDotcomContactPage() ? <ContactPageLink />
+    : <a href={contributionsEmail[props.countryGroupId]}>contact us here</a>;
```
Comment (jfsoul) [inline support-frontend/assets/components/legal/contribLegal/contribLegal.jsx]: nitpick and maybe not really our decision, but this is the one place we use `contact us here` instead of just `contact us`

File: support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala (hunk 1 above)
Comment (rupertbates) [reply support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala]: It's related to the issue with a rogue lambda updating all the product rate plan ids on UAT which broke an integration test and stopped me pushing this branch. This made it easier to debug so I want to leave it in

File: support-frontend/assets/components/legal/contribLegal/contribLegal.jsx (hunk 2 above)
Comment (rupertbates) [reply support-frontend/assets/components/legal/contribLegal/contribLegal.jsx]: Yeah that is true but I put it down to being more of an inconsistency than a deliberate decision. Happy to change if you think it's worth it?

Comment (jfsoul) [reply support-frontend/assets/components/legal/contribLegal/contribLegal.jsx]: I agree that it's probably not deliberate, I would change it

File: support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala (hunk 1 above)
Comment (jfsoul) [reply support-services/src/test/scala/com/gu/support/catalog/CatalogServiceIntegrationSpec.scala]: Ok, makes sense

File: support-frontend/assets/pages/paper-subscription-landing/components/content/helpers.jsx [hunk 3]
```diff
@@ -66,25 +70,32 @@ const ContentHelpBlock = ({
 }: {|
   faqLink: Element<string>,
   telephoneLink: Element<string>
-|}) => (
-  <Content appearance="feature" modifierClasses={['faqs']}>
-    {flashSaleIsActive('Paper', GBPCountries) &&
+|}) => {
+
+  const contactUs = useDotcomContactPage() ? <ContactPageLink /> :
+  <span>call our customer services team on {telephoneLink}</span>;
```
[the remark this answers is not in the document]
Comment (rupertbates) [reply support-frontend/assets/pages/paper-subscription-landing/components/content/helpers.jsx]: ES Lint disagrees! 😄
